packages/libs/react-ui/src/components/Form/ComboBox/index.ts

```diff
@@ -0,0 +1 @@
+export { ComboBox, type IComboBoxProps } from './ComboBox';
```

Comment:
The team decided to only exporting `Field` components moving forward that utilize the react-aria hooks since they add aria props to labels, error messages, etc. 

The PR for the `TextField` component has not been merged yet so it is probably better to wait until that one is completed to apply it to this component, but would be good to add a TODO to add the `Field` props in the future.